utah-design-system-website/src/@types/jsDocTypes.d.js

```diff
@@ -401,6 +407,7 @@
  *  @property {string} title
  *  @property {PageUrl[]} [parentLinks]
  *  @property {WebsiteMainMenuItem[]} [children]
+ *  @property {'inline' | 'flyout' | 'mega-menu' | 'plain' | null} [childrenMenuType]
```

Comment:
This could import and use the header's `ChildrenMenuTypes`
`{ChildrenMenuTypes | null} [childrenMenuType]`